src/fuzzy.js

```diff
@@ -299,8 +299,9 @@ function sellersShouldContinue(node, _, term, threshold, sValue, lastValue) {
 	return 1 - (bestPossibleValue / term.length) >= threshold;
 }
 
-// recursively walk the trie
+// recursively walk the tree
```

Comment:
This wasn't actually a typo.
A trie is a specific kind of tree, although I would say the name being so close is rather unfortunate:
https://en.wikipedia.org/wiki/Trie

Reply:
ok, fixed it